mysql/charset.go

```diff
@@ -15,6 +15,25 @@ package mysql
 
 import "unicode"
 
+// CharsetNameToID maps charset name to its default collation ID.
+func CharsetNameToID(charset string) uint8 {
+	// Use quick path for TiDB to avoid access CharsetIDs map
+	// "SHOW CHARACTER SET;" to see all the supported character sets.
+	if charset == "utf8mb4" {
+		return 45
```

Comment:
Perhaps define consts for these 5 constants so we don't need to repeat the special values in CharsetIDs again.

Reply:
add in https://github.com/pingcap/parser/pull/248/commits/9093fbb5c65f9dd6a9ec5968ed23c387cebca438

Reply:
I mean the constants 45, 63, 33, 11, 8 😅